ui/src/styles/index.scss

```diff
@@ -200,6 +200,10 @@
   width: 16px;
 }
 
+.ant-result-subtitle {
```

Comment:
This is part of your [other](https://github.com/0xPolygonID/sh-id-platform/pull/236) PR?

Reply:
Yep, I've cherry-picked the same commit.

Reply:
Why though? What do you need it for here?

Reply:
To properly render the multiline env parsing errors:

<img width="880" alt="image" src="https://user-images.githubusercontent.com/15896177/226416668-8903fff1-facf-411a-8603-248ded75531a.png">

Reply:
My question is: why is this duplicated in this PR?

Reply:
Because otherwise in this PR we would see this, which is incorrect:

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/15896177/226420707-794e017c-4a8c-4148-ba66-b45591de3084.png">